datumaro/util/test_utils.py

```diff
@@ -40,13 +33,11 @@ def __init__(self, path, is_dir=False):
     def __enter__(self):
         return self.path
 
-    # pylint: disable=redefined-builtin
-    def __exit__(self, type=None, value=None, traceback=None):
+    def __exit__(self, *args, **kwargs):
```

Comment:
I would suggest to use the names from the [documentation](https://docs.python.org/3/reference/datamodel.html#object.__exit__): `exc_type`, `exc_value`, `traceback`.

Reply:
Fixed